### ISSUE TITLE
Add an arrow head for quiver plots

### ISSUE THREAD

**Issue by bugalo** [ppinard/matplotlib-scalebar#37]
Would it be possible to add an arrow head to the scalebar? Then it could be used with quiver plots.

**Comment by ppinard** [comment]
I'm not so familiar with quiver plots. Is it simply that the scale bar has an arrow at the end?

![image](https://user-images.githubusercontent.com/715280/106358794-44ca6300-6306-11eb-870a-333786d84591.png)

**Comment by bugalo** [comment]
Yes, it is exactly that. This would be useful, for example, if you have an image with a given scalebar, and a quiver plot overlayed on top with its own "scalearrow".
